app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,30 @@
+$Event-color-background: Theme-color(background, default)
+$Event-color-borderTop: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
+$Event-height: 150px
+$Event-spacing-base: $Theme-spacing-base
+$Event-text-title-color: Theme-color(text, default)
+$Event-width: 280px
+
+.Event
+  background-color: $Event-color-background
+  border-radius: 3px
+  border-top: 5px solid $Event-color-borderTop
+  box-shadow: 0 2px 10px -3px gray
```

Comment:
I'm still not happy with the fact that you're inserting every background you think of into that bag. But whenever you have your final design language it will be easier to get to something better